### ISSUE TITLE
test/win failure: Test_getcompletion() on AppVeyor

### ISSUE THREAD

**Issue by erw7** [neovim/neovim#10649]
The test added in #10616 fails on AppVeyor like the following log.

https://ci.appveyor.com/project/neovim/neovim/builds/26299493/job/qksju85jlp3tactt?fullLog=true#L15999

This test also succeeds on Windows if it is `set shellslash`. We `set shellslash` in unix.vim as follows, why does AppVeyor fail this test? This test succeeds in my environment.

https://github.com/neovim/neovim/blob/3cf4b0d5e577f147f25c8417e097374c7ab88618/src/nvim/testdir/unix.vim#L8

**Comment by blueyed** [comment]
I think we should use shellslash like Vim there (https://github.com/neovim/neovim/pull/10197).
(See also the discussion referenced from there in https://github.com/neovim/neovim/pull/10189#issuecomment-501296559.)
I've rebased #10197, so you could / we can compare the output from before/after.

I think what needs to be done there then is also removing any extra handling of this setting in the tests themselves (to follow upstream/Vim there).

**Comment by erw7** [comment]
`-u unix.vim` set to `RUN_VIMTEST` seems to be overwritten by `-u NONE` when passing to `runnvim.sh`. I do not know if this is what was intended. But `unix.vim` is not used now, so I think it is safe to delete. Either way, it seems like we need to organize our settings.

https://github.com/neovim/neovim/blob/3cf4b0d5e577f147f25c8417e097374c7ab88618/src/nvim/testdir/Makefile#L148

https://github.com/neovim/neovim/blob/3cf4b0d5e577f147f25c8417e097374c7ab88618/src/nvim/testdir/Makefile#L161

**Comment by blueyed** [comment]
It looks like it is rather wrong that we're using it not anymore - Vim appears to skip it only with `opt_test.vim` and `test_gui.vim`.  Created https://github.com/neovim/neovim/pull/10650 for that.

**Comment by blueyed** [comment]
The following patches are missing from the Makefile:
```
v8.0.1179
v8.1.0424
v8.1.0456
v8.1.0723
v8.1.0724
4c92e75dd4ddb68dd92a86dd02d53c70dd4af33a
c854898881c02259f90170f210bf65bbd20cc1fc
v8.1.0967
v8.1.1476
v8.1.1478
v8.1.1479
v8.1.1505
v8.1.1637
v8.1.1694
```
(might be more, given that https://github.com/neovim/neovim/pull/10650 was marked N/A)

**Comment by blueyed** [comment]
NOTE: the shellslash was commented in https://github.com/neovim/neovim/commit/79a8d905ab8908362109284240d2f9a75564f8ad, which also brought some other diff to setup.vim.

**Comment by blueyed** [comment]
Note that Vim uses `dos.vim` actually for MingW also.. https://github.com/blueyed/vim/blob/tests-noswapfile/src/testdir/dos.vim

**Comment by erw7** [comment]
Closed because it was fixed in #10558.
